src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -14,12 +13,12 @@ internal static class PathUtilities
     {
         // We consider '/' a directory separator on Unix like systems. 
         // On Windows both / and \ are equally accepted.
-        internal static readonly char DirectorySeparatorChar = IsUnixLikePlatform ? '/' : '\\';
+        internal static readonly char DirectorySeparatorChar = PlatformInformation.IsUnix ? '/' : '\\';
```

Comment:
This is odd - perhaps we should just use Path.DirectorySeparatorChar instead of defining our own constant? I think this was done this way before we moved to netstandard1.3.